elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -376,10 +378,10 @@ default AuditLogger getAuditLogger() {
      */
     default DataStore getDataStore(MetaDataStore metaDataStore, AggregationDataStore aggregationDataStore,
             EntityManagerFactory entityManagerFactory) {
-
+	CancelSession cancelSession = new CancelSession(entityManagerFactory.get().unwrap(Session.class));
```

Comment:
YOu can make this a local variable and use it multiple places.

Reply:
Another issue - you are unwrapping the factory - not the entity manager.  I don't think that is correct.